community/sample-apps/gatsby-contentful-auth0/app/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
@praveenweb Is it okay if we remove this file? We already have a top-level license.

Reply:
Sure, can be removed.